feature/bgp/addpath/ate_tests/addpath/README.md

```diff
@@ -0,0 +1,89 @@
+
+* Configure ATE1:Port1 and ATE2:Port2 to advertise same prefix "prefix-1" with different path attributes to DUT:Port1 and DUT:Port2
+  * Verify that the DUT is advertising multiple paths to prefix-1 to RRCs ATE3 and ATE4 as well as to the RRS ATE5 with different path-ids
+* Configure RRCs ATE3:Port3 and ATE4:Port4 to advertise "prefix-2" to DUT with different Path attributes.
+  * Verify that the DUT advertises multiple paths for prefix-2 to ATE5 with different path-ids
+
+**Test-2**: Verify ADDPATH Receive capability 
+* DUT:Port3 and DUT:Port4 has IBGP peering with directly connected ATE3:Port3 and ATE4:Port4 respectively. In this case, DUT is the RR server and ATE3 and ATE4 are RR clients
+* DUT:Port5 has IBGP peering with directly connected ATE5:Port5. In this case DUT is the RR client and ATE5:Port5 is the RR server
+* Ensure that the EBGP and IBGP peering are enabled for both address famailies.
+* DUT is configued with addpath Receive capability only on its peering with ATE3:Port3 and ATE4:Port4
+  * During BGP capabilities negotiation in Open message, verify that the DUT negotiated addpath cability with Send/Receive field set to "1" signaling Receive capability.
+ * DUT is configued with addpath Send capability only on its peering with ATE5:Port5
+  * During BGP capabilities negotiation in Open message, verify that the DUT negotiated addpath cability with Send/Receive field set to "2" signaling send capability.
+ * Enable BGP multipath as well in the DUT.
+ * ATE3, 4 and 5 should be conffigured for addpath "both" capability.
+ * Configue ATE3 and ATE4 each to advertise 1M v4 prefixes with 32 different NHs for each prefix with a different path-id. Requirement here is for the DUT to receive 1M routes with 64 different NHs for the same prefixes but different path-id. Follow the same process for IPv6 with 600k routes and 32 distict NHs for each prefix.
+  * Among the advertised v4 prefixes, 500k routes should be ECMP routes for some set of NHs (advertised to the DUT w/ same path attributes). For same 500k routes, there should also be non-best routes that can be tested as receive and installed as UCMP (set of NHs advertised w/ different path attributes).
+  * Follow the same procedure as above for 300k (out of total 600k) v6 routes.
+  * Veirfy on ATE5 that it receives all v4 and v6 routes by checking for the received prefix count. However, for certain set of prefixes (example 500 v4 and 500 v6 prefixes), ensure that they are received with 64 different NHs each with different path-ids.
+ * Withdraw 500k v4 prefixes and 300k v6 prefixes from ATE4 and verify the following for the withdrawn prefixes
+  * ATE5 now has routes for the withdrawn prefixes only with 32 path-ids originated by ATE3
+  * For the withdrawn routes, ensure that the DUT installed the prefixes in its FIB with only 32 NHs generated by ATE3
+   
+**Test-3**: Test addpath configuration at different levels
```

Comment:
```suggestion
Test addpath configuration at different levels
```
This is not a TestCase. This is just iteration instruction, which will result in 6 Test cases in total

Reply:
Thanks for highlighting. I moved this instruction to the top.